app/views/content/life-as-a-teacher/explore-subjects/biology/_article.html.erb

```diff
@@ -1,44 +1,31 @@
-      <p>Biology is the study of living things, the intricate systems that sustain them, and the delicate balance of our ecosystems. As a biology teacher, you'll have the opportunity to inspire the next generation to explore the wonders of life itself.</p>
-      <p>In addition to the personal rewards of teaching biology, there are practical benefits too, including:</p>
-      <ul>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">generous and secure pension</a></li>
-        <li>more holiday days than people in many other professions</li>
-        <li>opportunities for progression</li>
-      </ul>
+      <p><%= @front_matter["subject"].titleize %> is the study of living things, the intricate systems that sustain them, and the delicate balance of our ecosystems. As a biology teacher, you'll have the opportunity to inspire the next generation to explore the wonders of life itself.</p>
+      <%= render 'content/shared/teaching/benefits' %>
     </div>
   </section>
 
-  <section class="col col-720">
-    <%= render Content::QuoteComponent.new(
-    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
-    name: "Chris Farr, teacher training adviser",
-    large: true
-  ) %>
+    <section class="col col-720">
+    <!-- Quote: Chris Farr -->
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
   </section>
 
   <section class="col col-720 col-space-l-top">
     <h2 class="heading--box-blue">Shape your pupils' futures</h2>
-      <p>Teaching biology opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from zoology and wildlife conservation, to medicine and dentistry.</p>
-      <p>By solving challenging problems, your pupils could make significant contributions to these impactful areas.</p>
-      <p>Your pupils will learn more from you than you might think.  As their teacher, you'll guide them though the skills they need to show their understanding of biology. You'll teach them valuable life skills such as critical thinking, decision making and communication.</p>
+      <p>Teaching <%= @front_matter["subject"] %> opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from zoology and wildlife conservation, to medicine and dentistry.</p>
+      <p>Your pupils will learn more from you than you might think.  As their teacher, you'll guide them though the skills they need to show their understanding of <%= @front_matter["subject"] %>. You'll teach them valuable life skills such as critical thinking, decision making and communication.</p>
```

Comment:
This is missing the `By solving challenging problems......` sentence

This is a very generic sentence - should this and the sentence after `Your pupils will learn....` be made into a new partial? However, I haven't checked all the other subject pages to see if it would work elsewhere (only the chemistry one - which would need updating as well if you add a new partial)

Will leave you to decide @joegibb @Sarah-DfE !

Reply:
I think this was a partial originally, but then we noticed that we've changed the text for other subjects e.g. languages,

Reply:
In which case, should we just add the `By solving challenging problems.....` sentence directly on this page then?